yt_dlp/extractor/imgur.py

```diff
@@ -76,6 +76,24 @@ class ImgurIE(ImgurBaseIE):
             'thumbnail': 'https://i.imgur.com/jxBXAMCh.jpg',
             'dislike_count': int,
         },
+    },
+        # from https://github.com/yt-dlp/yt-dlp/issues/9458
+        {
+        'url': 'https://imgur.com/zV03bd5',
+        'add_ies': ['Imgur'],
```

Comment:
don't need `add_ies`, but can you add `md5` field please?

Reply:
added the md5